src/Symfony/Component/Messenger/Tests/EventListener/SendFailedMessageForRetryListenerTest.php

```diff
@@ -56,7 +57,9 @@ public function testIsRetryableFalseCausesNoRetry()
         $sender->expects($this->never())->method('send');
         $sendersLocator = new Container();
         $sendersLocator->set('my_receiver', $sender);
-        $listener = new SendFailedMessageForRetryListener($sendersLocator, new Container());
+        $retryStrategyLocator = new Container();
+        $retryStrategyLocator->set('my_receiver', new MultiplierRetryStrategy(0));
+        $listener = new SendFailedMessageForRetryListener($sendersLocator, $retryStrategyLocator);
```

Comment:
without this change we would only test that a message is not retried when no retry strategy can be found (but that's already tested in the previous `testNoRetryStrategyCausesNoRetry()` method)

Reply:
No assert needs to be updated/added? Feels like the test is brittle if this was missing and nothing else needs an update